### PR TITLE
Add doc about usage stats tracking

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
In this PR I'm adding a documentation page to explain which information are tracked by the anonymous usage stats reporting.

#### Which issue(s) this PR fixes or relates to

Part of #1815

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by colega — APPROVED** [review]
Technical side LGTM, let's see what docs squad has to say :)

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Grafana Mimir includes a system to anonymously report non-sensitive and non-personal identifiable information about the running Mimir cluster to a remote statistics server.
+Mimir maintainers use these anonymous information to learn more about how the opensource community runs Mimir and what the Mimir team should focus on when working on the next features and documentation improvements.
+
+The anonymous usage statistics reporting is **disabled by default**.
+If possible, we ask you to enable the usage statistics.
+
+## The statistics server
+
+When the usage statistics reporting is enabled, the information is collected by a server run by Grafana Labs and exposed at `https://stats.grafana.org`.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:18]:
```suggestion
When usage statistics reporting is enabled, information is collected by a server that Grafana Labs runs. Statistics are exposed at `https://stats.grafana.org`.
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Which information is collected
+
+When the usage statistics reporting is enabled, Grafana Mimir collects the following information:
+
+- Information about the **Mimir cluster and version**
+  - Unique randomly-generated Mimir cluster identifier (e.g. `3749b5e2-b727-4107-95ae-172abac27496`).
+  - Timestamp when the anonymous usage statistics reporting was enabled the first time and the cluster identifier was created.
+  - The Mimir version (e.g. `2.3.0`).
+  - The Mimir branch, revision and golang version used to build the binary.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:28]:
```suggestion
- Information about the **Mimir cluster and version**:
  - A unique, randomly-generated Mimir cluster identifier, such as `3749b5e2-b727-4107-95ae-172abac27496`.
  - The timestamp when the anonymous usage statistics reporting was enabled for the first time, and the cluster identifier that was created that that time.
  - The Mimir version, such as `2.3.0`.
  - The Mimir branch, revision and golang version that was used to build the binary.
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- Information about the **environment** where Mimir is running
+  - The operating system (e.g. `linux`) and architecture (e.g. `amd64`) where Mimir is running.
+  - The Mimir memory utilization and number of goroutines.
+  - The number of logical CPU cores available to the Mimir process.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:32]:
```suggestion
- Information about the **environment** where Mimir is running:
  - The operating system, such as `linux`.
  - The architecture, such as `amd64`.
  - The Mimir memory utilization and number of goroutines.
  - The number of logical CPU cores available to the Mimir process.
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- Information about the Mimir **configuration**
+  - The `-target` parameter value (e.g. `all` when running Mimir in monolithic mode).
+  - The `-blocks-storage.backend` value (e.g. `s3`).
+  - The `-ingester.ring.replication-factor` value (e.g. `3`).
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:36]:
```suggestion
- Information about the Mimir **configuration**:
  - The `-target` parameter value, such as `all` when running Mimir in monolithic mode.
  - The `-blocks-storage.backend` value, such as `s3`.
  - The `-ingester.ring.replication-factor` value, such as `3`.
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  - Querier:
+    - The number of requests to queriers, split by API endpoint type (no information is tracked about the actual request or query). The following endpoints are tracked:
+      - Remote read.
+      - Instant query.
+      - Range query.
+      - Exemplars query.
+      - Labels query.
+      - Series query.
+      - Metadata query.
+      - Cardinality analysis query.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:51]:
```suggestion
- Information about the Mimir **cluster scale**:
  - Ingester:
    - The number of in-memory series.
    - The number of tenants that have in-memory series.
    - The number of samples and exemplars ingested.
  - Querier, *where no information is tracked about the actual request or query*:
    - The number of requests to queriers that are split by API endpoint type:
      - Remote read.
      - Instant query.
      - Range query.
      - Exemplars query.
      - Labels query.
      - Series query.
      - Metadata query.
      - Cardinality analysis query.
```
By "split", do you mean "categorized"?

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+> **Note**: Mimir maintainers commit to keep the list of tracked information updated over time and report any change both in the CHANGELOG and release notes.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:53]:
```suggestion
> **Note**: Mimir maintainers commit to keeping the list of tracked information updated over time, and reporting any change both via the CHANGELOG and the release notes.
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+## How to enable the anonymous usage statistics reporting
+
+If you would like to participate in usage statistics reporting, the feature can be enabled setting the CLI flag `-usage-stats.enabled=true` or the following YAML configuration:
+
+```yaml
+usage_stats:
+  enabled: true
+```
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:62]:
```suggestion
## Enable the anonymous usage statistics reporting

If you would like to participate in usage statistics reporting, set the CLI flag `-usage-stats.enabled=true` or change the following YAML configuration:

```yaml
usage_stats:
  enabled: true
```
```

Comment (osg-grafana) [reply docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:62]: I think this last suggestion got borked on line 62.

**Review by osg-grafana — APPROVED** [review]
Unblocking with feedback.

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,7 +6,7 @@
 
 * [CHANGE] Distributor: if forwarding rules are used to forward samples, exemplars are now removed from the request. #2710 #2725
 * [CHANGE] Limits: change the default value of `max_global_series_per_metric` limit to `0` (disabled). Setting this limit by default does not provide much benefit because series are sharded by all labels. #2714
-* [FEATURE] Introduced an experimental anonymous usage statistics tracking (disabled by default), to help Mimir maintainers driving better decisions to support the opensource community. The tracking system anonymously collects non-sensitive and non-personal identifiable information about the running Mimir cluster, and is disabled by default. #2643 #2662 #2685 #2732
+* [FEATURE] Introduced an experimental anonymous usage statistics tracking (disabled by default), to help Mimir maintainers driving better decisions to support the opensource community. The tracking system anonymously collects non-sensitive and non-personal identifiable information about the running Mimir cluster, and is disabled by default. #2643 #2662 #2685 #2732 #2733
```
Comment (osg-grafana) [inline CHANGELOG.md:9]:
```suggestion
* [FEATURE] Introduced an experimental anonymous usage statistics tracking (disabled by default), to help Mimir maintainers make better decisions to support the open source community. The tracking system anonymously collects non-sensitive, non-personally identifiable information about the running Mimir cluster, and is disabled by default. #2643 #2662 #2685 #2732 #2733
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+      - Labels query.
+      - Series query.
+      - Metadata query.
+      - Cardinality analysis query.
+
+> **Note**: Mimir maintainers commit to keep the list of tracked information updated over time and report any change both in the CHANGELOG and release notes.
+
+## How to enable the anonymous usage statistics reporting
+
+If you would like to participate in usage statistics reporting, the feature can be enabled setting the CLI flag `-usage-stats.enabled=true` or the following YAML configuration:
```
Comment (pracucci) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:57]:
```suggestion
## Enable the anonymous usage statistics reporting

If you would like to participate in usage statistics reporting, set the CLI flag `-usage-stats.enabled=true` or change the following YAML configuration:
```

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md (hunk 7 above)
Comment (pracucci) [reply docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:62]: I picked the same suggestion without editing YAML here:
https://github.com/grafana/mimir/pull/2733/files#r955743546

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+- Information about the Mimir **configuration**:
+  - The `-target` parameter value, such as `all` when running Mimir in monolithic mode.
+  - The `-blocks-storage.backend` value, such as `s3`.
+  - The `-ingester.ring.replication-factor` value, such as `3`.
+- Information about the Mimir **cluster scale**:
+  - Ingester:
+    - The number of in-memory series.
+    - The number of tenants that have in-memory series.
+    - The number of samples and exemplars ingested.
+  - Querier, *where no information is tracked about the actual request or query*:
```
Comment (pracucci) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:43]:
```suggestion
  - Querier, _where no information is tracked about the actual request or query_:
```

**Pull request merged**